### DISCUSSION TITLE
알림 어떻게 할까요?

### DISCUSSION THREAD

**Discussion by laigasus** [Strong-Potato/TripVote-BE#4] [General]
논의가 필요합니다!

**Comment by Nine-JH** [comment]
어디까지 논의가 되신건가요?

**Comment by laigasus** [comment]
@Nine-JH 아직 진척된 것이 없습니다.
안계시는 동안  Discussion 사용법 예시를 보이고자 제가 이 글을 올렸습니다

**Reply by Nine-JH** [reply]
아 그렇군요 ㅋㅋ discussion 좋네요

**Comment by laigasus** [comment]
@Nine-JH 저희끼리 주말에 잠깐 이야기했을 때는
SSE + Redis(pub/sub) 로 일단 먼저 해보자는 이야기가 나왔어요!

다른 분들도 유용한 정보나 기술 있으면 올려주셔도 좋을 것 같아요!

**Reply by Dr-KoKo** [reply]
프론트에서 구현하는 것은 큰 무리 없어 보입니다.
https://jellybeanz.medium.com/server-sent-events-server-push-%EB%9E%80-da6775de5c5d
https://developer.mozilla.org/ko/docs/Web/API/Server-sent_events/Using_server-sent_events
